pulsar-broker/src/main/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentTopic.java

```diff
@@ -484,14 +485,22 @@ private CompletableFuture<Void> delete(boolean failIfHasSubscriptions, boolean c
         return deleteFuture;
     }
 
+
+    @Override
+    public CompletableFuture<Void> close(boolean closeWithoutWaitingClientDisconnect) {
+        return close(false, closeWithoutWaitingClientDisconnect);
+    }
+
     /**
      * Close this topic - close all producers and subscriptions associated with this topic.
      *
+     * @param closeWithoutDisconnectingClients don't disconnect clients
```

Comment:
nit: I think the parameter should be `disconnectingClients`, it will be clearer to other developers. But it is not this PR's responsibility.

Reply:
yes, I was following the `closeWithoutWaitingClientDisconnect`'s convention. Probably we need a minor PR to fix this naming, if we want to refactor it.

Reply:
+1

Reply:
updated.